vue/src/vue-elements/settings-tab-panel.vue

```diff
@@ -62,6 +62,17 @@
             />
           </div>
         </div>
+        
+        <div
+          v-if="!isPro && generalSettings.default_interval <= 12"
```

Comment:
I think this should `< 12` or `<= 11` since 12 is the lowest available limit for Free.

Reply:
Done! Please review my latest commit.